crates/consensus/beacon/src/engine/mod.rs

```diff
@@ -1257,15 +1258,18 @@ where
         debug_assert!(self.sync.is_pipeline_idle(), "pipeline must be idle");
 
         let block_hash = block.hash;
+        let sw = Stopwatch::start_new();
         let status = self
             .blockchain
             .insert_block_without_senders(block.clone(), BlockValidationKind::Exhaustive)?;
+        let elapsed = sw.elapsed();
```

Comment:
We can probably just use the std lib time primitives here rather than importing a new crate, for example using:
```rust
let start = Instant::now();
// ... thing you want to time
let elapsed = start.elapsed();
```